### ISSUE TITLE
Typography size inconsistencies

### ISSUE THREAD

**Issue by maya** [uswds/uswds-site#745]
Using the web inspector to view the font size, many of them did not match up with the actual values on the docs site and fractal site with the written values.

<img width="1248" alt="Screen Shot 2019-03-28 at 12 30 31 PM" src="https://user-images.githubusercontent.com/5249443/55187120-6def9a80-5155-11e9-8571-ca674e7ec23d.png">

**Comment by thisisdano** [comment]
Hm. Let's see what's up here...

**Comment by thisisdano** [comment]
Ugh, I think this another instance of "site defaults do not match system defaults`.

The system theme font size defaults are:
```
$theme-type-scale-3xs:      2;
$theme-type-scale-2xs:      3;
$theme-type-scale-xs:       4;
$theme-type-scale-sm:       5;
$theme-type-scale-md:       6;
$theme-type-scale-lg:       9;
$theme-type-scale-xl:       12;
$theme-type-scale-2xl:      14;
$theme-type-scale-3xl:      15;
```

but the site defaults are
```
$theme-type-scale-3xs:      2;
$theme-type-scale-2xs:      3;
$theme-type-scale-xs:       4;
$theme-type-scale-sm:       5;
$theme-type-scale-md:       6;
$theme-type-scale-lg:       8; // different from system default
$theme-type-scale-xl:       11; // different from system default
$theme-type-scale-2xl:      14;
$theme-type-scale-3xl:      15;
```

Additionally, the site data lists the default tokens as something different from wither of these. So, first, I'll update the site data to match the system defaults — then work out something where we can get all of them in sync.

**Comment by maya** [comment]
Some size issues on the fractal site which shouldn't be affected by theme stuff: https://federalist-proxy.app.cloud.gov/preview/uswds/uswds/release-2.0/components/preview/typesetting.html

h3 (`font-serif-lg`) listed as 23.4px, actually is 21.4px
h5 listed as 14.6px, no near equivalent in docs
h6 (`font-serif-2xs`) is listed as 13.7px, actually is 13.8px 

The rest seem accurate.

**Comment by thisisdano** [comment]
Once the docs change is in the `h` values should be in sync:
h1: 2xl
h2: xl
h3: lg
h4: sm
h5: xs

**Comment by maya** [comment]
What are h6 and body font (p)? 2xs and xs?

**Comment by thisisdano** [comment]
`p` is probably set to `sm` (16px) — but I think in most cases it takes the normalized equivalent of 1rem.

From typography settings:
```
$theme-h1-font-size:               '2xl';
$theme-h2-font-size:               'xl';
$theme-h3-font-size:               'lg';
$theme-h4-font-size:               'sm';
$theme-h5-font-size:               'xs';
$theme-h6-font-size:               '3xs';
```

**Comment by thisisdano** [comment]
So, I don't think we ever explicitly declare a size for `p`. `body` will get either nothing (by default, via `$theme-respect-user-font-size`) or whatever is in `$theme-body-font-size` (if respect user font size is set to false').

**Comment by thisisdano** [comment]
It seems like we might be missing something wrt normalization and body font size, but I'd need to test it again — it's been too long since developing it.

**Comment by thisisdano** [comment]
Ah yes, and `.usa-prose` is a normalized container to whatever font face is set as the body font. SO I think it's all mostly working as expected.

**Comment by maya** [comment]
Fixed in #746.

**Comment by ameshkin** [comment]
We are upgrading to USWDS 3 and we're having an issue with the tokens for header sizes. They are compiled into css but the browser will say "invalid property value".  "    font-size: x-large;" works, but not the actual uswds tokens.  What are we doing wrong?

$theme-h1-font-size:               '2xl';
```
.field_heading {
    font-size: 2xl;
}
```
